Improvement: Added Readme for mesh handle tutorials, wiki links still missing. (#2372)
DLR-AMR/t8code: tutorials/README.md

A tutorial about the generation of curved adaptive meshes.

## Mesh handle
[Mesh Handle Tutorials](../tutorials/mesh_handle/README.md)

> These tutorials can be followed instead of the original t8code tutorials. They explain working with t8code with the help of the 

Reviewer: ```suggestion
These tutorials can be followed instead of the general t8code tutorials. They explain working with t8code with the help of the 
```
One idea is that the mesh handle is the only entry point of t8code which is why we should not name this "original"

[mesh_handle](../mesh_handle/README.md) interface. 

> Please note, that the mesh handle tutorials start at step 2, because steps 0 and 1 are the same as in the general tutorials. Please complete these first and then come back to the [Mesh Handle Tutorial Overview](../tutorials/mesh_handle/README.md)

Reviewer: ```suggestion
Please note, that the mesh handle tutorials start at step 2, because steps 0 and 1 are the same as in the general tutorials. Please complete these first and then come back to the [mesh handle tutorials](../tutorials/mesh_handle/README.md)
```

> Please note, that the mesh handle tutorials start at step 2, because the first 2 steps are the same as in the original tutorials. Please complete these first and then come back to the [Mesh Handle Tutorial Overview](../tutorials/mesh_handle/README.md)

Reviewer: ```suggestion
Please note, that the mesh handle tutorials start at step 2, because the steps 0 and 1 are the same as in the general tutorials. Please complete these first and then come back to the [Mesh Handle Tutorial Overview](../tutorials/mesh_handle/README.md).
```

A tutorial about the generation of curved adaptive meshes.

## Mesh handle
[Mesh Handle Tutorials](../tutorials/mesh_handle/README.md)

> These tutorials can be followed instead of the general t8code tutorials. They explain working with t8code with the help of the 

Reviewer: ```suggestion
The [mesh handle tutorials](../tutorials/mesh_handle/README.md) can be followed instead of the general t8code tutorials. They explain working with t8code with the help of the 
```
This looks werid in the preview as it is not interpreted as a title. Maybe like this?

[mesh_handle](../mesh_handle/README.md) interface. 

> Please note, that the mesh handle tutorials start at step 2, because steps 0 and 1 are the same as in the general tutorials. Please complete these first and then come back to the [Mesh Handle Tutorial Overview](../tutorials/mesh_handle/README.md)

Reviewer: I am confused, i thought you corrected this? Did you revert it somewhere?

> Please note, that the mesh handle tutorials start at step 2, because steps 0 and 1 are the same as in the general tutorials. Please complete these first and then come back to the [Mesh Handle Tutorial Overview](../tutorials/mesh_handle/README.md)

Reviewer: ```suggestion
Please note, that the mesh handle tutorials start at step 2, because steps 0 and 1 are the same as in the general tutorials. Please complete these first and then come back to the [mesh handle tutorial overview](../tutorials/mesh_handle/README.md).
```
